clients/java/src/main/java/io/zeebe/client/impl/worker/JobPoller.java

```diff
@@ -105,32 +114,20 @@ public void onError(final Throwable throwable) {
     if (retryPredicate.test(throwable)) {
       poll();
     } else {
-      try {
-        if (openSupplier.getAsBoolean()) {
-          logError(throwable);
+      if (openSupplier.getAsBoolean()) {
+        try {
+          LOG.warn(
+              "Failed to activated jobs for worker {} and job type {}",
+              requestBuilder.getWorker(),
+              requestBuilder.getType(),
+              throwable);
+        } finally {
+          errorCallback.accept(throwable);
         }
-      } finally {
-        pollingDone();
       }
     }
   }
 
-  private void logError(final Throwable throwable) {
```

Comment:
To be fair, we only log the backoff (delay for next retry) at debug level. I think it is important for clients to know that no new jobs are activated and this might be something that requires their attention, even with an automatic backoff mechanism. Logging this at warning level seems appropriate to me.